community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: single
+title:  "Release Manager Runbook"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+## Friday before the release window opens
+
+1. Ping #dev reminding everyone to merge outstanding changes
+by Monday:
+
+> I'll be cutting the $VERSION release branches on Tuesday,
+> so if there are any outstanding PRs that you'd like to get into $VERSION,
+> please make sure they are merged by EOD Monday. Once the branch is cut, only
+> fixes will be accepted into the release branches.
+
+## First Monday of the release window
+
+1. Ping #dev reminding everyone to merge outstanding changes ASAP:
+
+> I'll be cutting the $VERSION release branches tomorrow morning, so if
+> there are any outstanding PRs that you'd like to get into $VERSION, please make
+> sure they are merged ASAP. Once the branch is cut, only fixes will be accepted
+> into the release branches.
+
+## First Tuesday of the release window
+
+1. Reach out to anyone who has previously contacted you to ensure their
+last-minute release PRs have been merged.
+
+2. Merge all [kork version autobump PRs](https://github.com/issues?q=is%3Aopen+is%3Apr+org%3Aspinnaker+archived%3Afalse+-repo%3Aspinnaker%2Fspinnaker.github.io+-repo%3Aspinnaker%2Froer+-repo%3Aspinnaker%2Fkeel+-repo%3Aspinnaker%2Fdeck-customized+-repo%3Aspinnaker%2Fswabbie+-repo%3Aspinnaker%2Fstyleguide+-repo%3Aspinnaker%2Fdcd-spec+label%3Aautobump-kork)
```

Comment:
I think this URL will work better: https://github.com/pulls?q=is%3Apr+author%3Aspinnakerbot+is%3Aopen

And the text should probably be updated since there are many more besides kork and they should automerge, so any open ones by definition had a problem that needs to be fixed. So maybe "If there are any outstanding autobump PRs, make the required fixes to allow them to merge." Or something?

Helpfully, in creating that search, I found a rosco version bump that was just lying around and fixed it :)